inst/examples/chat.R

```diff
@@ -0,0 +1,79 @@
+library(shiny)
+library(purrr)
+library(dplyr)
+library(magrittr)
+
+ui <- shinyUI(fluidPage(
+  tags$head(
+    tags$style(HTML("
```

Comment:
Same as js code below. Please extract stylesheet and add it as a static file. This will greatly improve readability of this example (+ make it shorter!).